packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -187,7 +187,7 @@
     transform: translateY(-50%) scale(.75);
```

Comment:
This seems to be a duplicate of the selector just above it.

Reply:
:(